Games/types/Mafia/Action.js

```diff
@@ -93,8 +94,12 @@ module.exports = class MafiaAction extends Action {
       }
 
       for (let target of toCheck) {
-        if (target === player && !action.hasLabel("hidden")) {
-          visitors.push(action.actor);
+        if (
+          action.actor &&
+          action.actor instanceof Player &&
```

Comment:
in what case would actor not be a player? adding this line will only mask an incorrect use of actor